### PR TITLE
Pro 4801 is indeterminate

### PR DESCRIPTION
**Pull request by haroun**
## Summary

Remove annotate permission with `create`. If it's a new document, it will not be present in the DB and the annotate `create` won't be available.

Is indeterminate on checkboxes is needed to be able to tell is all checkboxes on the row are checked.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*

- [x] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/doc-type/index.js [hunk 1]
```diff
@@ -1870,7 +1870,6 @@ module.exports = {
           after(results) {
             // In all cases we mark the docs with ._edit and ._publish if
             // the req is permitted to do those things
-            self.apos.permission.annotate(query.req, 'create', results);
```
Comment (boutell) [inline modules/@apostrophecms/doc-type/index.js:1873]: What about pages? For pages, `create` permission on the parent page is requires to create a page.

Comment (boutell) [reply modules/@apostrophecms/doc-type/index.js:1873]: Unless that is handled outside of this PR?

**Review by boutell — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/doc-type/index.js (hunk 1 above)
Comment (haroun) [reply modules/@apostrophecms/doc-type/index.js:1873]: I'm checking with Val.

**Comment by boutell** [comment]
There shouldn't be any harm in keeping this annotation, it'll make it easier if we get around to folders for general purpose use in pieces, but do what works as long as pages behave sensibly

**Review by boutell — DISMISSED** [review]

**Review by ValJed — APPROVED** [review]

**Pull request merged**